tinytuya/scanner.py

```diff
@@ -164,6 +164,7 @@ def __init__( self, ip, deviceinfo, options, debug ):
         self.timeo = 0
         self.resets = 0
         self.step = FSCAN_NOT_STARTED
+        self.try_v35 = False
```

Comment:
Hey @uzlonewolf is this really only v35 or v34_and_v35? I'm trying to follow the logic/flow it seems like both, but I'm probably missing something.

Reply:
It's only looked at when the device version is set to 3.4, so I guess `try_v35_with_v34` would be a more accurate name.

Reply:
Got it! Yes, that makes sense. Thanks @uzlonewolf !